pyidxp/aws/dynamodb.py

```diff
@@ -36,3 +36,14 @@ def get_table(self, table_name):
         while table.describe()['Table']['TableStatus'] != 'ACTIVE':
             sleep(1)
         return table
+
+    def update_table(self, table, throughput):
+        if self.conn.host == 'localhost':
+            return
+        self.is_table_active(table)
+        table.update(throughput=throughput)
+        self.is_table_active(table)
+
+    def is_table_active(self, table):
```

Comment:
I don't really liked this method's name... I think it should be something like `wait_until_table_is_active`. What do you think?